data.sql

```diff
@@ -1,6 +1,9 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals VALUES (1,'Agumon', '2020-FEB-03', 0, TRUE, 10.23);
-INSERT INTO animals VALUES (2,'Gabumon', '2018-NOV-15', 2, TRUE, 8);
-INSERT INTO animals VALUES (3,'Pikachu', '2021-JAN-07', 1, FALSE, 15.04);
-INSERT INTO animals VALUES (4,'Devimon', '2017-MAY-12', 5, TRUE, 11);
+INSERT INTO animals VALUES (1,'Charmender', '2020-Feb-08', 0, false, -11);
+INSERT INTO animals VALUES (2,'Plantom', '2021-Nov-15', 2, true, -5.7);
+INSERT INTO animals VALUES (3,'Squirtle', '1993-Apr-02', 3, false, -12.13);
+INSERT INTO animals VALUES (4,'Angemon', '2005-Jun-12', 1, true, -45);
+INSERT INTO animals VALUES (5,'Boarmon', '2005-Jun-07', 7, true, 20.4);
+INSERT INTO animals VALUES (6,'Blossom', '1998-Oct-13', 3, true, 17);
+INSERT INTO animals VALUES (7,'Ditto', '2022-May-14', 4, true, 22);
```

Comment:
- [ ] Kindly you are not supposed to remove your previous data. This milestone is the continuation of the previous one. Kindly add your previous data.

Reply:
Thanks. I have implemented it.